deployment/stacks/service/overlays/dev/template/uec-dos-api-cs-service-customisation.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: uec-dos-api-cs-proxy
+  namespace: K8S_APP_NAMESPACE_TO_REPLACE
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-ssl-cert: arn:aws:acm:eu-west-2:AWS_ACCOUNT_ID_NONPROD_TO_REPLACE:certificate/c0718115-4e22-4f48-a4aa-8c16ea86c5e6
```

Comment:
This certificate has been replaced in the base overlay by: alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:eu-west-2:AWS_ACCOUNT_ID_TO_REPLACE:certificate/AWS_CERTIFICATE_TO_REPLACE. Should something similar happen here?